Dockerfile

```diff
@@ -1,8 +1,16 @@
-FROM golang:1.8.5-jessie
+FROM golang:alpine AS build
 # create a working directory
 WORKDIR /go/src/app
+#COPY go.mod go.sum ./
+#RUN go mod download
 # add source code
 COPY . .
 # run main.go
```

Comment:
GCO_ENABLED=0 is needed here too, or the program won't build with C or C++ source files on root, which are need by main.go anyway.